### PR TITLE
Add settings

### PR DESCRIPTION
**Pull request by GoreevArtem**

### CODE REVIEW

**Comment by GoreevArtem** [comment]
@DumDereDum review pls

File: src/main.py [hunk 1]
```diff
@@ -21,12 +21,14 @@ def main():
...
     cap = cv.VideoCapture(0)
-    pgame = Game(cap, spath)
+
     monitor = get_monitors()
     m_width = monitor[0].width
     m_height = monitor[0].height
     while cap.isOpened():
         success, img = cap.read()
+        height, width = img.shape[:2]
+        pgame = Game(height, width, spath)
```
Comment (DumDereDum) [inline src/main.py]: relocate it back, because there you create a new class object every frame

File: src/samples/Sample_UI.py [hunk 2]
```diff
@@ -110,55 +226,41 @@ def display_video_stream(self):
         if not ret:
             return False
 
-        if not self.video:
-            img = cv.flip(img, turn)
+        # if not self.video:
+        height, width = img.shape[:2]
+        pgame = Game(height, width, spath)
```
Comment (DumDereDum) [inline src/samples/Sample_UI.py]: the same

**Review by DumDereDum — COMMENTED** [review]
create a new function in `class Piano` for changing settings
name it something like `re_init` or `set_settings` or anything else

File: src/main.py (hunk 1 above)
Comment (GoreevArtem) [reply src/main.py]: I have a problem with video playback. the piano shrank with him. this is the only way I have found

Comment (DumDereDum) [reply src/main.py]: try to create a variable `is_settings_changed`
then type this redefinition:
```
if is_settings_changed
    pgame = Game(height, width, spath)
```
but in main.py you mast to move this string out of `while`

**Comment by GoreevArtem** [comment]
@DumDereDum review pls

**Comment by DumDereDum** [comment]
There is infinit cycle after closing menu windows. It needs to be finded and fixed.

**Comment by GoreevArtem** [comment]
you mean the video selection?

**Comment by GoreevArtem** [comment]
@DumDereDum review pls

**Comment by GoreevArtem** [comment]
вернул выбор видео, уменьшается пианино

**Pull request merged**